Pipfile

```diff
@@ -65,6 +65,7 @@ braintree = "==3.49.0"
 razorpay = ">=1.1.1"
 stripe = "*"
 "urllib3" = "*"
+dataclasses = "*"
```

Comment:
Would it make sense to limit this to install only on 3.6? This package is not needed on 3.7 after all.

Reply:
Good idea

Reply:
Done during rebasing, thanks!